build.gradle.kts

```diff
@@ -56,24 +63,38 @@ sourceSets {
 
 dependencies {
     // create a build config for every plugin-release-module combination
-    for (pluginConfig in pluginConfigs) {
-        val type = pluginConfig.type
+    for (pluginConfig in pluginConfigs.skipWebModules()) {
         val release = pluginConfig.release
         val config = pluginConfig.name
-
-        // common dependencies
-        config(kotlin("stdlib"))
-        config(kotlin("test"))
-        config(kotlin("test-junit"))
-        config("io.ktor:ktor-$type-core:$release")
-        when(type) {
-            "client" -> config("io.ktor:ktor-client-mock:$release")
-            "server" -> config("io.ktor:ktor-server-test-host:$release")
+        when(pluginConfig.module) {
+            "web" -> {
```

Comment:
would be nice to have an enum

Reply:
I went with strings here with the though of introducing custom modules in the future.  I don't think that'll be any time soon, so I'll replace with an enum for now 👍